snowtrick/public/js/detail.js

```diff
@@ -0,0 +1,83 @@
+document.addEventListener('DOMContentLoaded', () => {
+
+        // add event on trigger button
+    function activeTrashButton() {
+        const trash_btn = document.querySelectorAll(".trash_btn");
+        trash_btn.forEach((trash) => {
+            trash.addEventListener('click', (event) => {
+                let myInit = {
+                    method: trash.data.method,
+                    headers: {'content-type': 'application/json'}
```

Comment:
Codacy has a fix for the issue: [Replace `··········headers:·{'content-type':·'application/json'}` with `headers:·{·"content-type":·"application/json"·},`](https://app.codacy.com/gh/trousse/OC_Projet6/pullRequest?prid=10057695)
```suggestion
          headers: { "content-type": "application/json" },
```